setup.cfg

```diff
@@ -1,10 +1,11 @@
 [flake8]
 min-version = 2.7
 accept-encodings = utf-8
+extend-exclude = test
 # errors we ignore; see https://www.flake8rules.com/ for more info
 ignore =
     # pycodestyle errors
-    E121,  # continuation line under-indented for hanging indent
+#    E121,  # continuation line under-indented for hanging indent
```

Comment:
I think it makes sense to just remove them all together instead of keeping them around as a comment

Reply:
I agree removing them for final merge, but as noted below the tests still need to be changed. 

Opening PR to start discussion.

I can get tests done tonight if no objections.